src/components/Input.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Button from './Button'
+import { nanoid } from 'nanoid'
+
+function Input({ onCreateCard }) {
```

Comment:
I think it makes sense to rename the component, because right now it implies that it is just an input field (like the button in the Button.js), so maybe something like FormQuestion?